### DISCUSSION TITLE
[Bug report] doc bug in post config options for Image upload sample code

### DISCUSSION THREAD

**Discussion by memcpy-rand-rand-rand** [GetPublii/Publii#1824] [Bug Report]
### Operating system

N/A

### Publii version

N/A

### Issue type

Something else

### Bug description

https://getpublii.com/dev/post-config-options/

In Image upload section, first sample code snippet says:

"postConfig": [
        {
            "name": "displayImage",
            "label": "Upload image",
            "value": "",
            "type": "image"
        },

the second and third snippets do not reference the right name for the image:

 <figure>
   <img
      src="{{@config.post.**displayCustomFeaturedImage**}}"
      {{imageDimensions @config.post.**displayCustomFeaturedImage**}}>
</figure>

they should be "displayImage", not "displayCustomFeaturedImage" to align with the post config.

**Comment by bobmitro** [comment]
Thank you for indicating the issue. It has been fixed.
